portal/src/graphql/portal/ManageLanguageWidget.module.css

```diff
@@ -10,7 +10,7 @@
 }
 
 .searchBox {
-  width: 300px;
+  max-width: 300px;
```

Comment:
The widget should have its label on the left, and its control on the right in the mobile view.